website/plugins/og-plugin/README.md

```
@@ -1 +1 @@
-The code is taken from [signalwire docs](https://github.com/signalwire/docs/tree/eb2ff7ee3a6029edea4b0a1643ef65a05a195f88/website/src/plugins/docusaurus-plugin-og).
\ No newline at end of file
+The code is taken from [signalwire docs](https://github.com/signalwire/docs/tree/eb2ff7ee3a6029edea4b0a1643ef65a05a195f88/website/src/plugins/docusaurus-plugin-og).
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)